OWNERS

```diff
@@ -2,9 +2,11 @@ approvers:
 - anguslees
 - NickrenREN
 - dims
+- Fedosin
```

Comment:
Mike, let's add you as a reviewer first and then move to owner later. ok?

Reply:
Sure! Done.